app/models/macro.rb

```diff
@@ -21,7 +21,7 @@ class Macro < ApplicationRecord
 
   belongs_to :account
   belongs_to :created_by,
-             class_name: :User, optional: true
+             class_name: :User, optional: true, inverse_of: :macros
```

Comment:
I'm seeing the usage of inverse_of multiple places. Why do we need these?

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html

```
The automatic guessing of the inverse association uses a heuristic based on the name of the class, so it may not work for all associations, especially the ones with non-standard names
```

ref: https://stackoverflow.com/a/39670478/939299

```
 It's a hint to ActiveRecord to use already loaded data instead of fetching it again through a relationship.
 ```